index.bs

```diff
@@ -479,7 +479,7 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
     1. [=list/Append=] |result| to |clientExtensions|.
 
-1. Let |clientData| be a new {{ClientData}} instance whose fields are:
+1. Let |clientData| be a new {{CollectedClientData}} instance whose fields are:
```

Comment:
this is a nit, but it seems the WhatWG/W3C style is to say..
`Let |collectedClientData| be a new {{CollectedClientData}} instance...`